tools/update-rc3.in

```diff
@@ -982,6 +981,7 @@ download_osimage() (
 
     [ -s "${aTGT_FILE}" ] && [ -s "${aTGT_FILE_CSOLD}" ] && [ -s "${aTGT_FILE_MANIFEST}" ] && \
     touch -r "${aTGT_FILE}" "${aTGT_FILE_CSOLD}" "${aTGT_FILE_MANIFEST}" && \
+    ( etn-sw-update --add-image "${aTGT_FILE}" || echo "$?" > "/var/lib/fty/sw-update/activation_requested" ) && \
```

Comment:
Is there any `>&2` to log into that file too?

Reply:
no, we just want the return code of --add-image, which is here the index of the added image... hence the ||